package.json

```diff
@@ -134,8 +134,8 @@
     "tmp": "0.0.28"
   },
   "scripts": {
-    "feature-test": "./bin/cucumber.js -p build",
-    "feature-test-es5": "./bin/cucumber.js -p build -p es5",
+    "feature-test": "node ./bin/cucumber.js -p build",
```

Comment:
just checking in on this. So did running `npm run feature-test` not work on windows without this change?